### PR TITLE
reorder guide to put emphasis on developing/verifying with convox start first

### PR DESCRIPTION
**Pull request by mattmanning**
## Release Playbook
- [x] Rebase against master
- [x] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by mattmanning** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;yrZjyBYj&#x2F;294-getting-started-guide-oriented-towards-convox-start"> getting started guide oriented towards convox start</a>

File: _introduction/getting-started.md [hunk 1]
```diff
@@ -52,22 +41,9 @@ You can [download the CLI package](https://dl.equinox.io/convox/convox/stable) o
...
-
-Finally you need to log in with the API key found **Account** section in the top navigation bar:
-
-    $ convox login
-    Password: <Your Console API key>
-
 ## Run an Application Locally
 
-`convox start` helps your team develop sophistated applications locally. We maintain a set of sample applications that demonstrate a feature-rich development environment with:
+`convox start` helps your team develop sophistated applications locally. We maintain a set of [sample applications](https://github.com/convox-examples) that demonstrate a feature-rich development environment with:
```
Comment (mwarkentin) [inline _introduction/getting-started.md]: `sophistated` -> `sophisticated`

File: _introduction/getting-started.md [hunk 2]
```diff
@@ -84,11 +60,37 @@ Finally you need to log in with the API key found **Account** section in the top
 
     $ convox start
 
+The application will be available at the address of your Docker host (`localhost` when using Docker for Mac). Once an application can be successfully booted with `convox start` it is ready to be deployed. The following steps will guide you through installing a Rack and deploying your application to it.
+
+## Sign Up
+
+You first need to sign up to the Convox Console, a web UI for managing your Racks, Organizations and Integrations.
```
Comment (mwarkentin) [inline _introduction/getting-started.md]: Maybe link directly to the sign up page inline here?

File: _introduction/getting-started.md [hunk 3]
```diff
@@ -84,11 +60,37 @@ Finally you need to log in with the API key found **Account** section in the top
...
+* Visit the Console [signup page](https://console.convox.com/grid/signup)
+* Create an account
+
+### Create an Organization
+
+You start out in a `personal` organization where the Racks you install are only visible to you. If you'd like to create a deployment environment that is shared with other members of your team, click on the **personal** dropdown followed by **Create Organization**. After this you can invite members of your team and assign them roles that control what they can access.
+
+## Install a Rack
+
+Click on **Add a Rack** followed by **Install a New Rack** in the top navigation bar. For the name enter `production` if you plan to deploy production services, or `staging` or `development` if this is for testing.
```
Comment (mwarkentin) [inline _introduction/getting-started.md]: I know this was existing copy, but the `, or .. or ..` seems a bit awkward - maybe change to ".., or `staging`/`development` if this is for testing."

**Comment by mattmanning** [comment]
@mwarkentin Thanks for the suggestions.

**Pull request merged**